src/Codeception/Lib/Driver/Facebook.php

```diff
@@ -30,83 +38,96 @@ public function __construct($config, $logCallback = null)
      */
     public function createTestUser($name, array $permissions)
     {
+        $app_token = $this->appId . '|' . $this->appSecret;
```

Comment:
since Codeception is using PSR-2 coding standard, please fix the code accordingly, thanks